_includes/03-links.md

```diff
@@ -0,0 +1 @@
+Replace this with a link to your a GitHub or social media profile.
```

Comment:
Now, you're going to add an image. You can add an image of yourself or anything else you'd like to feature. You'll also learn how to create descriptive text, or "alt text", for images, and how to create links.
Images
Let's add an image. Don't forget to include descriptive text in the square brackets. This text is read aloud for people using screen readers. It's also shown at times when your image doesn't display, such as when there's a poor connection. You can see the syntax for images below:

![GitHub Logo](/images/logo.png)